release_notes/48-release-notes.adoc

```diff
@@ -46,44 +54,159 @@ This release adds improvements related to the following components and concepts:
+
+Starting with {product-title-short} 4.8, the system now reports both the CVE ID (Common Vulnerabilities and Exposures) and the RHSA (Red Hat Security Advisory) when available. RHSAs might include one or more security fixes, and might also contain bug or enhancement updates. In previous versions up to {product-title-short} 4.7, {product-title-short} replaced the CVE ID with the corresponding RHSA ID once Red Hat released a fix for the associated vulnerability.
+
+
+[id="notable-technical-changes_{context}"]
 == Notable technical changes
 
 This release contains the following changes:
 
+//ROX-26577
+* Starting with {product-title-short} 4.8, Scanner V4 is the default scanner for reporting vulnerabilities in User Workloads, Platforms, and Nodes for all new installations of {product-title-short} Central and Secured Clusters.
+
+* {product-title-short} 4.8 preserves the current scanner configuration for existing deployments that you upgrade. If you're using the StackRox Scanner, it remains in use after the upgrade. For switching to Scanner V4, see xref:../operating/examine-images-for-vulnerabilities.adoc#enabling_scanner_v4_examine-images-for-vulnerabilities[Enabling Scanner V4].
+
+* Scanner V4 runs in Central and you do not have to deploy it to secured clusters unless you have specific requirements, for example:
+** Accessing image registries that are not reachable from Central.
+** Using the {ocp} image registry.
+** Running on {product-title-short} Cloud Service with firewall restrictions that limit registry access to internal traffic.
+** Using registry mirroring.
++
+For more details, see xref:../operating/examine-images-for-vulnerabilities.adoc#accessing-delegated-image-scanning_examine-images-for-vulnerabilities[Accessing delegated image scanning].
+
+* In `roxctl` CLI, certificate validation failures are now marked as errors.
+
+* {product-title-short} 4.8 includes the updated `roxctl` help command output making it more readable. The output is now more consistent with other command-line tools.
+
+* Red Hat has moved the `SecurityPolicy`` Custom Resource Definition (CRD) to the template directory within the Helm chart. This change simplifies CRD maintenance if you are using Helm, as it will now automatically upgrade.
++
+[IMPORTANT]
+====
+If you are using Helm to manage your {product-title-short} installation, you must apply the following changes to the `SecurityPolicy` CRD before upgrading to avoid upgrade failures:
+[source,terminal]
+----
+$ kubectl annotate crd/securitypolicies.config.stackrox.io meta.helm.sh/release-name=stackrox-central-services <1>
+$ kubectl annotate crd/securitypolicies.config.stackrox.io meta.helm.sh/release-namespace=stackrox <2>
+$ kubectl label crd/securitypolicies.config.stackrox.io app.kubernetes.io/managed-by=Helm
+----
+<1> Update the `release-name` annotation to match the name if you used different name during your initial installation. The default value is `stackrox-central-services`.
```

Comment:
For this footnote, and the next, put the "if" statement first. This makes it easier for users to decide if they need to read the rest of the sentence.